README.md

```diff
@@ -451,6 +451,28 @@ az storage account keys list \
 
 * Install the app using your values.
 
+### Deploying on a new cluster for testing purposes
+
+You might find yourself in a situation where you want to deploy loki on a new cluster for testing purposes only. Depending on the testing requirements, you might need or want to avoid to create object storage with a cloud-provider and manage its access permissions for your loki pods.
```

Comment:
```suggestion
You might find yourself in a situation where you want to deploy Loki on a new cluster for testing purposes only. Depending on the testing requirements, you might need to avoid creating an object storage with a cloud-provider and manage its access permissions for your Loki pods.
```